packages/sd-jwt-vc/package.json

```diff
@@ -35,11 +35,13 @@
   },
   "license": "Apache-2.0",
   "dependencies": {
-    "@sd-jwt/core": "workspace:*"
+    "@sd-jwt/core": "workspace:*",
+    "jwt-status-list": "^0.0.4"
```

Comment:
Love the addition! I think to do a full review of this we should also review the `jwt-status-list` repository. 

Is that something you would also consider donating to this repository? `@sd-jwt/status-list` would work quite well, even if it is not fully sd-jwt specific.

Also, I see that the `jwt-status-list` has a direct dependency on `jose`, which IIRC does not work in React Native. Due to this, Credo would not be able to use this library anymore. I think we should either remove the `jose` dependency or make the status list functionality plugable and the user providers the dependency for their use case.

Reply:
I mentioned the "home" of the `jwt-status-list` in the OWF discord channel. I would be fine to move it for now to the monorepo of `@sd-jwt` since it is primarily designed for this.

The requirement of JOSE should be removed when it's blocking the integration of react native. But before removing it and try to implement everything on our own I would love to get the feedback that jose is not compatible with react native (since jose has no dependencies)

Reply:
> I mentioned the "home" of the `jwt-status-list` in the OWF discord channel. I would be fine to move it for now to the monorepo of `@sd-jwt` since it is primarily designed for this.

Perfect!

> The requirement of JOSE should be removed when it's blocking the integration of react native. But before removing it and try to implement everything on our own I would love to get the feedback that jose is not compatible with react native (since jose has no dependencies)

There are only [two runtimes](https://github.com/panva/jose/tree/2595451b67772dfc82231cddd43050293857139f/src/runtime) (browser and nodejs), which both use node.js or browser specific APIs. This could be fixed with a bunch of polyfills, but I think that the status list library does not need to do any heavy JWT functionality and can be implemented manually. Does not have to be in this PR, but we would need to do it before integrating it within Credo. I also would not mind helping with it.

Reply:
@berendsliedrecht I published version 0.0.5 without the dep of `jose`, it was moved to dev deps for the testing.

Reply:
Also agree with Berend, it would be great to not depend on jose, and keep everything platform/runtime agnostic. Polyfills for crypto are the biggest pain of RN.

If the implementation is moved to this repo, we can reuse the crypto callbacks used for signing/verifying SD-JWT VC?

Reply:
The creation of the signed jwt is out of scope of the library. You get the payload and header and then decide yourself how you want to issue the jwt that holds the status list. For the verification the callbacks are provided + added tests